argilla/src/argilla/_models/_settings/_metadata.py

```diff
@@ -22,19 +21,12 @@
 from argilla._models import ResourceModel
 
 
-class MetadataPropertyType(str, Enum):
-    terms = "terms"
-    integer = "integer"
-    float = "float"
-
-
 class BaseMetadataPropertySettings(BaseModel):
-    type: MetadataPropertyType
     visible_for_annotators: Optional[bool] = True
 
 
 class TermsMetadataPropertySettings(BaseMetadataPropertySettings):
-    type: Literal[MetadataPropertyType.terms]
+    type: Literal["terms"] = "terms"
```

Comment:
Is necessary to define a default? Will something like the following works?:

```suggestion
    type: Literal["terms"]
```

Reply:
Yes. We need to define a default. Otherwise, the model expects a literal value for the attribute. 
https://github.com/pydantic/pydantic/issues/6346